examples/threathunter/threat_intelligence/threatintel.py

```diff
@@ -41,13 +43,13 @@ def push_to_cb(self, feed_id, results):
                     "severity": int(result.severity),
                     "iocs_v2": [ioc_v2.as_dict() for ioc_v2 in result.iocs_v2]
                 }
-
-                if self.input_validation([report_dict]):
+                try:
+                    ReportSchema.validate(report_dict)
```

Comment:
This function returns the object after a successful validation though this seems functional too. Just wanted to double check that this doesn't modify the report_dict object

Reply:
I don't think the validation modifies the report dictionary, as far as I can see in testing